static/app/utils/useDevToolbar.tsx

```diff
@@ -25,10 +25,12 @@ export default function useDevToolbar({enabled}: {enabled: boolean}) {
         SentrySDK: Sentry,
         apiPrefix,
         environment: ['prod'],
-        organizationSlug: 'sentry',
-        projectId: 11276,
+        organizationSlug:
+          process.env.NODE_ENV !== 'development' ? 'sentry' : 'sentry-test',
+        projectId: process.env.NODE_ENV !== 'development' ? 11276 : 5270453,
         projectPlatform: 'javascript',
-        projectSlug: 'javascript',
+        projectSlug:
+          process.env.NODE_ENV !== 'development' ? 'javascript' : 'app-frontend',
```

Comment:
Should this be in here?

Reply:
While this is PoC we are hard coding the org/project slugs, this allows the links to work in dev mode (assuming the replay integration is enabled)

Reply:
can you test with `dev-ui`. my workflow depends on that command

Reply:
It works, but you need to comment out this condition in useReplayInit: https://github.com/getsentry/getsentry/blob/2a1da081f3a9e4e4111b577d5551fa24691da374/static/getsentry/gsApp/utils/useReplayInit.tsx#L85-L87

I think it's disabled so we don't record everyone's dev-ui sessions. Maybe we could uncomment but set both sample rates to 0?

Reply:
I'll make a ticket and add it to the dev toolbar project

Reply:
probably hold off on a ticket to change `useReplayInit`. The toolbar should support whatever env it finds itself inside of. So if sometimes there's no `replayIntegration` at all, that should be ok. We'll be able to test that case in dev, and on prod there will be a `replayIntegration` so we can test that case as well!

Reply:
I wasn't thinking about the other panels, this change would break them for dev. I'm done testing, let's just leave this panel disabled in dev.